test/NJsonApiCore.Test/Serialization/JsonApiTransformerTest/TestLinkData.cs

```diff
@@ -0,0 +1,119 @@
+﻿using NJsonApi;
+using NJsonApi.Serialization.Documents;
+using NJsonApi.Serialization.Representations;
+using NJsonApi.Serialization.Representations.Resources;
+using NJsonApi.Test.Builders;
+using NJsonApi.Test.TestControllers;
+using System;
+using Xunit;
+using NJsonApi.Infrastructure;
+
+namespace NJsonApi.Test.Serialization.JsonApiTransformerTest
+{
+    public class TestLinkData
+    {
+        [Fact]
+        public void Creates_CompoundDocument_for_single_class_with_no_links_and_properly_map_no_links()
```

Comment:
Are we actually asserting the links in the resulting CompoundDocument or just checking it doesn't break?